tap_linear/streams.py

```diff
@@ -47,6 +51,12 @@ class TeamsStream(LinearStream):
     replication_key = "updatedAt"
     query = teamsQuery
 
+    def get_child_context(self, record: dict, context) -> dict:
+        """Return a context dictionary for child streams."""
+        return {
+            "variables":{"teamId": record["id"]}
```

Comment:
I would make the `teamId` top level in this object

Reply:
Done